### PR TITLE
Add migration guide to README

### PR DESCRIPTION
**Pull request by billy-addepar**

### CODE REVIEW

**Comment by billy-addepar** [comment]
@Addepar/ice

File: README.md [hunk 1]
```diff
@@ -117,3 +117,11 @@ By default Ember table cell renderes simple text defined inside each `column`. T
   width: 180
 }
 ```
+## Migrating from old Ember table
+To support smooth migration from old version of Ember table (support only till ember 1.11), we have move the old source code to separate package [ember-table-legacy](https://github.com/Addepar/ember-table-legacy). It's a separate package from this Ember table package and you can install it using yarn or npm. This allows you to have 2 versions of ember table in your code base and you can start your migrating one table at at time. The recommended migration steps is as follow (if you are using ember 1.11):
+
+1) Rename all your ember-table impot to ember-table-legacy. (for example: `import EmberTable from 'ember-table/components/ember-table'` becomes `import EmberTableLegacy from 'ember-table-legacy/components/ember-table-legacy'`. Remove reference of `ember-table` in `package.json`.
```
Comment (pzuraq) [inline README.md]: typo, `impot` -> `imports`

File: README.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 # Ember Table
 
 An addon to support large data set and a number of features around table. `Ember Table` can
-handle over 100,000 rows without and rendering or performance issue.
+handle over 100,000 rows without and rendering or performance issue. This version of `Ember Table` supports 1.11 to latest version of Ember.
```
Comment (pzuraq) [inline README.md]: We should add line breaks in paragraphs regularly rather than relying on word wrap

File: README.md [hunk 3]
```diff
@@ -107,7 +107,7 @@ If you want to use default table style, import the `ember-table/default` SASS fi
 To use footer for your table, pass `footerRows` param to ember table. Each element in `footerRows` represents a row in table footer. The footer row takes `valuePath` field in each column to render data for each footer cell, similar to table body.
 
 ### Custom header and custom footer
-By default Ember table cell renderes simple text defined inside each `column`. To custom table header or footer, you can pass in `headerComponent` and `footerComponent` fields in each column data.
+By default Ember table header renders text defined by `columnName` or `footerValue` inside each `column`. To custom table header or footer, you can pass in `headerComponent` and `footerComponent` fields in each column data. When the `headerComponent`(or `footerComponent`) is defined, the `columnName`(or `footerValue`) field is ignored.
```
Comment (pzuraq) [inline README.md]: To custom table header or footer -> to _customize the_ table header or footer

File: README.md [hunk 4]
```diff
@@ -117,3 +117,11 @@ By default Ember table cell renderes simple text defined inside each `column`. T
   width: 180
 }
 ```
+## Migrating from old Ember table
+To support smooth migration from old version of Ember table (support only till ember 1.11), we have move the old source code to separate package [ember-table-legacy](https://github.com/Addepar/ember-table-legacy). It's a separate package from this Ember table package and you can install it using yarn or npm. This allows you to have 2 versions of ember table in your code base and you can start your migrating one table at at time. The recommended migration steps is as follow (if you are using ember 1.11):
+
+1) Rename all your ember-table impot to ember-table-legacy. (for example: `import EmberTable from 'ember-table/components/ember-table'` becomes `import EmberTableLegacy from 'ember-table-legacy/components/ember-table-legacy'`. Remove reference of `ember-table` in `package.json`.
+2) Install `ember-table-legacy` using `yarn add ember-table-legacy` or `npm install ember-table-legacy`
+3) Run your app to make sure that it works without issue.
+4) Install this `ember-table` repo
```
Comment (pzuraq) [inline README.md]: Probably better to word this like "Reinstall the latest version of the main `ember-table` package"

File: README.md [hunk 5]
```diff
@@ -117,3 +117,11 @@ By default Ember table cell renderes simple text defined inside each `column`. T
   width: 180
 }
 ```
+## Migrating from old Ember table
+To support smooth migration from old version of Ember table (support only till ember 1.11), we have move the old source code to separate package [ember-table-legacy](https://github.com/Addepar/ember-table-legacy). It's a separate package from this Ember table package and you can install it using yarn or npm. This allows you to have 2 versions of ember table in your code base and you can start your migrating one table at at time. The recommended migration steps is as follow (if you are using ember 1.11):
```
Comment (pzuraq) [inline README.md]: "The recommended migration steps is as follow" -> "The recommended migration steps _are_ as _follows_"

**Pull request merged**